src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -226,12 +226,14 @@ protected override async Task CreateVersionsAsync(CancellationToken cancellation
             if (latestPrerelease.version != null
                 && (latestStableVersion.version == null || latestPrerelease.version > latestStableVersion.version))
             {
-                _versions.Add(new DisplayVersion(latestPrerelease.version, Resources.Version_LatestPrerelease, isDeprecated: latestPrerelease.isDeprecated));
+                var versionRange = new VersionRange(latestPrerelease.version, true, latestPrerelease.version, true);
+                _versions.Add(new DisplayVersion(versionRange, latestPrerelease.version, Resources.Version_LatestPrerelease, isDeprecated: latestPrerelease.isDeprecated, isVulnerable: latestPrerelease.isVulnerable));
```

Comment:
Wanted to avoid creating another constructor to `DisplayVersion`. [Creating a range](https://github.com/NuGet/NuGet.Client/blob/c4e43daa7a8884ac5b0c90b6e058408c5618ce4c/src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs#L27) like we do when using `NuGetVersion` https://github.com/NuGet/NuGet.Client/blob/c4e43daa7a8884ac5b0c90b6e058408c5618ce4c/src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs#L146

Reply:
At some point, maybe now, that class could use some refactoring (maybe to factory methods?). It's hard to know what all the different variations of a display version mean, when to use them, etc.